### ISSUE TITLE
CSS shorthand prevents component from getting server-side-rendered

### ISSUE THREAD

**Issue by levitanong** [fulcrologic/fulcro#199]
It appears that the CSS shorthand `:.some-class.another-class$this-too` prevents a component from getting server-side-rendered.
```clj
;; ui-something won't be html-stringified
(dom/div :.foo
  (ui-something))

;; works
(dom/div {:className "foo"}
  (ui-something))
```

**Comment by levitanong** [comment]
https://github.com/levitanong/repro-ssr

```
(in-ns 'repro-ssr.server)
(ssr)
```

see `repro-ssr.ui.root/Root`

**Comment by awkay** [comment]
Fixed in d678cc9f
